acceptable/openapi.py

```diff
@@ -21,17 +22,41 @@ def _to_dict(source: Any):
         for item in source:
             source_list.append(_to_dict(item))
         return source_list
+    elif hasattr(source, "__dict__"):
+        source_dict = {}
+        for key, value in source.__dict__.items():
+            source_dict[key] = _to_dict(value)
+        return source_dict
     else:
         return source
 
 
-def dump(metadata, stream):
+@dataclass
+class OasInfo(object):
+    description: str = ""
+    version: str = ""
+    title: str = ""
+    tags: list = field(default_factory=lambda: [])
+    contact: dict = field(default_factory=lambda: {"name": "", "email": ""})
+
+
+@dataclass
+class OasRoot31(object):
+    openapi: str = "3.1.0"
+    info: OasInfo = OasInfo()
+    servers: dict = field(default_factory=lambda: {})
+    paths: dict = field(default_factory=lambda: {})
+    components_schemas: dict = field(default_factory=lambda: {})
+
+
+def dump(metadata: APIMetadata, stream):
+    service_name = None
+    if len(metadata.services) == 1:
+        service_name = list(metadata.services.keys())[0]
 
-    # TODO: parse metadata as OpenAPI specification
+    oas = OasRoot31()
+    oas.info.title = service_name or ""
+    oas.info.version = metadata.current_version or ""
 
-    return yaml.safe_dump(
-        _to_dict(None),
-        stream,
-        default_flow_style=False,
-        encoding=None,
-    )
+    oas_to_dict = _to_dict(oas)
+    return yaml.safe_dump(oas_to_dict, stream, default_flow_style=False, encoding=None, )
```

Comment:
At first sight I thought `oas_to_dict` was a function. Maybe rename as `oas_dict`? Or just use `_to_dict(oas)` directly?
Oh, and mind the trailing comma.